packages/user-interface/src/components/formio/FormIoPasswordInput.tsx

```diff
@@ -0,0 +1,80 @@
+import { Components } from "@formio/react";
+import TextInput from "@gemeente-denhaag/text-input";
+import { ExtendedComponentSchema, Formio } from "formiojs";
+import { useId } from "react";
+import { Container } from "react-dom/client";
+import BasicFormIoComponentSchema from "./BasicFormIoComponentSchema";
+import FormField from "@gemeente-denhaag/form-field";
+import FormLabel from "@gemeente-denhaag/form-label";
+import useFormIoState, {
+  FormIoRefProp,
+  useFormIoStateProps,
+} from "./useFormIoState";
+import BaseFormIoComponent from "./BaseFormIoComponent";
+
+type FormIoPasswordInputProps = BasicFormIoComponentSchema &
+  useFormIoStateProps;
+
+const FormIoPasswordInput = ({
+  formioRef,
+  onChange,
+  disabled,
+  placeholder,
+  label,
+}: FormIoPasswordInputProps) => {
+  const [value, setValue] = useFormIoState({ formioRef, onChange });
+  const id = useId();
+
+  return (
+    <FormField>
+      <FormLabel htmlFor={id}>{label}</FormLabel>
+      <TextInput
+        id={id}
+        type="password"
+        value={value || ""}
+        disabled={disabled}
```

Comment:
```suggestion
        {...props}
```

Is dit nog een optie voor disabled en placeholder of maakt dat meer stuk dan goed?

Reply:
Geen idee haha, ik check het lokaal eerst wel even.

Reply:
Dit moeten we niet willen, er worden veel te veel ongeldige attributen in de input velden gespread hierdoor.